ceph/bootstrap-cluster.sh

```diff
@@ -1,14 +1,22 @@
 export PATH=/root/bin:$PATH
```

Comment:
Can you add shebang line: #!/bin/bash ?

Reply:
why? any problem with this?

Reply:
The #! shebang is used to tell the kernel which interpreter should be used to run the commands present in the file.
Although not mandatory, it's recommended (i.e. `#!/usr/bin/env bash`).

Reply:
added